reference/alerts.v1.yaml

```diff
@@ -143,7 +143,9 @@ components:
       content:
         'application/json':
           schema:
-            $ref: './common/models/alertsconfig.v1.yaml'
+            allOf:
+              - $ref: '#/components/schemas/alertsConfig'
```

Comment:
Should this be combined into the `alertsconfig.v1.yaml`? Is it only returned in a response?

Mainly curious since the other use of `alertsconfig.v1.yaml` in `UpsertConfiguration` does not include this `alertsConfig` model.

Reply:
A fair question. Perhaps some better names at least would be useful.

The distinction is intentional. as during an upsert, the properties of `./common/models/alertsconfig.v1.yaml` can be changed, but not the properties of `#/components/schemas/alertsConfig`---those are read-only.